var/spack/repos/builtin/packages/rpm/package.py

```diff
@@ -0,0 +1,136 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os
+from spack import *
+import llnl.util.tty as tty
+
+
+class Rpm(AutotoolsPackage):
+    """
+    The RPM package manager
+    https://github.com/rpm-software-management/rpm/blob/master/INSTALL
+    Note that debugedit is an additional tool that *should* compile with
+    adding libdw and elfutils, but the library (as of 4.16.1.2) is being
+    removed from rpm and will be put somewhere else.
+    """
+
+    homepage = "https://github.com/rpm-software-management/rpm"
+    url      = "https://github.com/rpm-software-management/rpm/archive/rpm-4.16.0-release.tar.gz"
+
+    version('4.16.0', sha256='a62b744e3404b107e8467e1a36ff0f2bf9e5c1b748dbfeb36db54bbb859446ea')
+    version('4.16.1.2', sha256='3d2807807a8ccaa92a8ced74e09b5bf5b2417a5bbf9bee4abc7c6aa497547bf3')
+
+    variant('openssl', default=False, description='use openssl for cryptographic library')
+    variant('sqlite', default=False, description='use sqlite instead of ndb')
+    variant('bdb_ro', default=False, description='standalone support for read-only BDB databases')
+    variant('selinux', default=False, description="enable support for SELinux")
+    variant('python', default=False, description="build Python bindings to RPM library")
+    variant('posix', default=False, description="enable POSIX.1e draft 15 file capabilities support")
+    variant('gpg', default=False, description="install gpg for using cryptographic signatures")
+    variant('openmp', default=True, description="OpenMP multithreading support (default).")
```

Comment:
Just a minor nitpick. Defaults are shown in `spack info` already, so they are not needed in the description:
```suggestion
    variant('openmp', default=True, description="OpenMP multithreading support")
```